classes/class-wc-connect-shipping-label.php

```diff
@@ -39,11 +54,22 @@ class WC_Connect_Shipping_Label {
 		public function __construct(
 			WC_Connect_API_Client $api_client,
 			WC_Connect_Service_Settings_Store $settings_store,
-			WC_Connect_Service_Schemas_Store $service_schemas_store
+			WC_Connect_Service_Schemas_Store $service_schemas_store,
+			WC_Connect_Payment_Methods_Store $payment_methods_store
 		) {
 			$this->api_client = $api_client;
 			$this->settings_store = $settings_store;
 			$this->service_schemas_store = $service_schemas_store;
+			$this->payment_methods_store = $payment_methods_store;
+			$this->account_settings = new WC_Connect_Account_Settings(
+					$settings_store,
+					$payment_methods_store
+			);
+			$this->package_settings = new WC_Connect_Package_Settings(
+					$settings_store,
+					$service_schemas_store
+			);
+			$this->continents = new WC_Connect_Continents();
```

Comment:
Next hack week let's setup [PHP-DI](http://php-di.org/) so we don't have to do all this stuff.

Reply:
Yeah, let's do that. I'm a big fan of dependency injection.